### ISSUE TITLE
Retrieve Spreadsheet Export with Release on Project List view

### ISSUE THREAD

**Issue by JaideepPalit** [eclipse/sw360#1060]
(from sw360 user)

The export of project list view's spreadsheet generation yields a 500 error

1. Start SW360
2. go to Projects
3. Tag „ SI DG SA&P" or any other narrow down"
4. Search
5. Export Spreadsheet
6. select option „with linked releases“

result on Chrome 500:

**Comment by JaideepPalit** [comment]
```
Caused by: java.lang.NoClassDefFoundError: org/apache/commons/compress/archivers/zip/ZipFile
	at org.eclipse.sw360.exporter.ExcelExporter.makeExcelExport(ExcelExporter.java:41)
	at org.eclipse.sw360.portal.portlets.projects.ProjectPortlet.exportExcel(ProjectPortlet.java:872)
	at org.eclipse.sw360.portal.portlets.projects.ProjectPortlet.serveResource(ProjectPortlet.java:207)
	at com.liferay.portlet.internal.FilterChainImpl.doFilter(FilterChainImpl.java:146)
	at com.liferay.portlet.ScriptDataPortletFilter.doFilter(ScriptDataPortletFilter.java:88)
	at com.liferay.portlet.internal.FilterChainImpl.doFilter(FilterChainImpl.java:140)
	at com.liferay.portal.kernel.portlet.PortletFilterUtil.doFilter(PortletFilterUtil.java:78)
	at com.liferay.portal.kernel.servlet.PortletServlet.service(PortletServlet.java:115)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.eclipse.equinox.http.servlet.internal.registration.EndpointRegistration.service(EndpointRegistration.java:153)
	at org.eclipse.equinox.http.servlet.internal.servlet.ResponseStateHandler.processRequest(ResponseStateHandler.java:62)
	at org.eclipse.equinox.http.servlet.internal.context.DispatchTargets.doDispatch(DispatchTargets.java:120)
	at org.eclipse.equinox.http.servlet.internal.servlet.RequestDispatcherAdaptor.forward(RequestDispatcherAdaptor.java:40)
	at com.liferay.portlet.internal.InvokerPortletImpl.invoke(InvokerPortletImpl.java:567)
	at com.liferay.portlet.internal.InvokerPortletImpl.invokeResource(InvokerPortletImpl.java:682)
	at com.liferay.portlet.internal.InvokerPortletImpl.serveResource(InvokerPortletImpl.java:496)
	at com.liferay.portal.monitoring.internal.portlet.MonitoringInvokerPortlet.serveResource(MonitoringInvokerPortlet.java:292)
	at com.liferay.portlet.internal.PortletContainerImpl._serveResource(PortletContainerImpl.java:1077)
	at com.liferay.portlet.internal.PortletContainerImpl.lambda$serveResource$4(PortletContainerImpl.java:267)
	at com.liferay.portlet.internal.PortletContainerImpl._preserveGroupIds(PortletContainerImpl.java:425)
	at com.liferay.portlet.internal.PortletContainerImpl.serveResource(PortletContainerImpl.java:260)
	at com.liferay.portlet.SecurityPortletContainerWrapper.serveResource(SecurityPortletContainerWrapper.java:208)
	at com.liferay.portlet.RestrictPortletContainerWrapper.serveResource(RestrictPortletContainerWrapper.java:150)
	at com.liferay.portal.kernel.portlet.PortletContainerUtil.serveResource(PortletContainerUtil.java:232)
	at com.liferay.portal.action.LayoutAction.processLayout(LayoutAction.java:389)
	at com.liferay.portal.action.LayoutAction.execute(LayoutAction.java:170)
	at com.liferay.portal.struts.PortalRequestProcessor._process(PortalRequestProcessor.java:414)
	at com.liferay.portal.struts.PortalRequestProcessor.process(PortalRequestProcessor.java:154)
	at com.liferay.portal.internal.servlet.MainServlet.doGet(MainServlet.java:204)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:634)
	at com.liferay.portal.internal.servlet.MainServlet.service(MainServlet.java:621)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:741)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:231)
	... 120 more
Caused by: java.lang.ClassNotFoundException: org.apache.commons.compress.archivers.zip.ZipFile cannot be found by sw360-exporters_1.0.0
	at org.eclipse.osgi.internal.loader.BundleLoader.findClassInternal(BundleLoader.java:508)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:419)
	at org.eclipse.osgi.internal.loader.BundleLoader.findClass(BundleLoader.java:411)
	at org.eclipse.osgi.internal.loader.ModuleClassLoader.loadClass(ModuleClassLoader.java:151)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:522)
	... 153 more
```

**Comment by mcjaeger** [comment]
fixed in #1061
